components/SelectFromTo.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+
+export function SelectFromTo() {
```

Comment:
I belive this should be better as slider, make it so you can pass (value,label)  pair array as props for SelectItem components